qemu/tests/virtio_port_hotplug.py

```diff
@@ -28,35 +29,48 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters.
     :param env:    Dictionary with test environment.
     """
-
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
+    os_type = params["os_type"]
     timeout = int(params.get("login_timeout", 360))
-    if params["os_type"] == "windows":
-        run_bg_test(test, params, vm)
-    for repeat in range(int(params.get("repeat_times", 1))):
-        repeat += 1
+    module = params.get("modprobe_module")
+    check_module = params.get("check_module", "yes")
+    session = vm.wait_for_login()
+    if os_type == "windows":
+        driver_name = params["driver_name"]
+        session = utils_test.qemu.windrv_check_running_verifier(
+            session, vm, test, driver_name)
+    else:
+        params['filesize'] = 2000
```

Comment:
define in cfg is ok.

Reply:
seems this is ok, cfg is the default value.

Reply:
It can work, but recommend to define in cfg for this case instead of update params in script

Reply:
do you mean this` params['filesize'] = params.get("filesize_linux", 2000)` , this seems a little redundant

Reply:
in cfg:
```
filesize = 100
Linux:
    filesize = 2000
```